ui-sdk-components-sample/app/src/main/java/com/karhoo/samples/networksdk/MainActivity.kt

```diff
@@ -0,0 +1,170 @@
+package com.karhoo.samples.networksdk
```

Comment:
the namespace, could be updated to `com.karhoo.samples.uisdk.components`

Reply:
Done